sfm_pc/templatetags/language_url.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from  more_itertools import unique_everseen
+
+from django import template
+from django.conf import settings
+from django.template.defaultfilters import stringfilter
+
+register = template.Library()
+
+@register.filter
+def create_select2_data(select_lang):
+  data_array = ["English", "French", "Spanish"]
+  trans_obj = {
+      'fr': 'French',
+      'en': 'English',
+      'es': 'Spanish',
+    }
+
+  full_text = trans_obj[select_lang]
+  data_array.insert(0, full_text)
+  data_array = list(unique_everseen(data_array))
```

Comment:
Interesting solution! I would have done this with list operations and a little functional programming instead of introducing `more-itertools` as a dependency:

```python
data_array.insert(0, data_array.pop(data_array.index(full_text))
```

But I'm pretty sure these are functionally identical, so it doesn't make much of a difference.

Reply:
That's cool! I did know that `pop` could accept an index number as an arg. The only problem if that `.index` finds the first instance of the element, and I'd like to preserve that. 

So, it would be possible to reverse the array, then remove the duplicate, and then reverse it again...what do you think? overkill?

Reply:
I think if the element we're after is at the start of the list, everything should still work fine. The code will wind up jumping through a hoop it doesn't actually need to jump through (by removing the first element and adding it to the start of the list, where it already was) but as I understand it that would still produce the right output. Does that make sense?

We could also add a check to avoid that situation altogether:

```python
if data_array[0] != full_text:
    data_array.insert(0, data_array.pop(data_array.index(full_text))
```

Reply:
Ah, yes...I like this check...and also, yes, I see how the `pop` can work, here (i.e., we insert after we pop).

Change forthcoming! (And happy to be rid of more-itertools.)